src/routes/library/+page.svelte

```diff
@@ -1,50 +1,106 @@
 <script lang="ts">
-  import marketplaceDecks from '$lib/data/marketplaceDecks';
+  import marketplaceDecks from '$lib/data/library/marketplaceDecks';
   import ProBadge from '$lib/components/ProBadge.svelte';
+  import LibrarySection from '$lib/components/LibrarySection.svelte';
+  import consonantsAndShortVowels from '$lib/data/library/playlists/consonantsAndShortVowels';
+  import consonantDigraphs from '$lib/data/library/playlists/consonantDigraphs';
+  import bannerImg from '$lib/assets/library-banner.png';
 </script>
 
 <svelte:head>
-  <title>Pro Decks - Blend</title>
+  <title>Blend Library</title>
   <meta name="description" content="Browse expertly crafted Blend decks, and import them into your Blend Pro account." />
 </svelte:head>
 
 <main>
-  <section class="library-description">
-    <div class="side-by-side heading">
-      <h1>Deck Library</h1>
-      <div class="badge"><ProBadge /></div>
+  <section class="banner">
+    <img src={bannerImg} alt="Blend Library Graphic" />
```

Comment:
Let's add placeholder width/height to this because there was noticeable content shift for me when it loaded